rayopt/simplex.py

```diff
@@ -192,5 +189,5 @@ def transform(self, t):
             p = simplex_transform(self.i.ravel(), self.j, self, t)
             return p.view(self.__class__)
 
-    Simplex.__name__ = str("Simplex{}d{}n".format(d0, n0))
+    Simplex.__name__ = str(f"Simplex{d0}d{n0}n")
```

Comment:
I don't remember why that `str()` was there. It seems weird.

Reply:
Removed.